src/app/auth/auth-registration/auth-registration.component.html

```diff
@@ -0,0 +1,35 @@
+<ion-header>
+  <ion-toolbar>
+    <ion-title>Registration</ion-title>
+  </ion-toolbar>
+</ion-header>
+<ion-content color="light" text-center>
+  <div class="div-logo registration">
+    <img src="/assets/logo.svg" width="60%">
+  </div>
+  <div class="div-after-logo">
+    <ng-container *ngIf="!hide_password">
+        <h4>Your Login Email Address</h4>
+        <p>{{user.email}}</p>
+    </ng-container>
+    <ng-container *ngIf="hide_password">
+        <h4>Your Mobile Number</h4>
+        <p>{{user.contact}}</p>
+    </ng-container>
+    <ion-list margin>
+      <ion-item>
+        <ion-input name="passowrd" [(ngModel)]="password" required type="password" placeholder="password"></ion-input>
+      </ion-item>
+      <ion-item>
+        <ion-input name="password" [(ngModel)]="rePassword" required type="password" placeholder="Conform Password"></ion-input>
+      </ion-item>
+      <ion-item lines="none">
+        <ion-checkbox color="primary" slot="start" [(ngModel)]="isAgreed"></ion-checkbox>
+        <ion-label class="terms-and-conditions">I agree to the Terms & Conditions</ion-label>
+      </ion-item>
+    </ion-list>
+    <p><a (click)="openLink()">Terms & Conditions</a></p>
+    <p>You need a password! You need to agree with terms and Conditions.</p>
+    <ion-button (click)="login()" shape="round" expand="block" color="primary">REGISTER</ion-button>
```

Comment:
Where is the `login()` in component? Also, since it is 'register', call the function `register()` makes more sense